packages/auto-label/src/auto-label.ts

```diff
@@ -225,36 +229,42 @@ handler.addLabeltoRepoAndIssue = async function addLabeltoRepoAndIssue(
   }
   const colorNumber =
     jsonArray?.findIndex((object: JSONData) => objectInJsonArray === object) %
```

Comment:
This statement is very hard to read.  Multiple split into a few lines so it's easier to follow?